x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -123,33 +126,34 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
     mutedInstanceIds: {
       type: 'keyword',
     },
-    meta: {
-      properties: {
-        versionApiKeyLastmodified: {
-          type: 'keyword',
-        },
-      },
-    },
+    // NO NEED TO BE INDEXED
+    // meta: {
+    //   properties: {
+    //     versionApiKeyLastmodified: {
+    //       type: 'keyword',
+    //     },
+    //   },
+    // },
     monitoring: {
       properties: {
         run: {
           properties: {
-            history: {
-              properties: {
-                duration: {
-                  type: 'long',
-                },
-                success: {
-                  type: 'boolean',
-                },
-                timestamp: {
-                  type: 'date',
-                },
-                outcome: {
-                  type: 'keyword',
-                },
-              },
-            },
+            // history: {
+            //   properties: {
+            //     duration: {
+            //       type: 'long',
+            //     },
+            //     success: {
+            //       type: 'boolean',
+            //     },
+            //     timestamp: {
+            //       type: 'date',
+            //     },
+            //     outcome: {
+            //       type: 'keyword',
+            //     },
+            //   },
+            // },
             calculated_metrics: {
```

Comment:
curious about this one. I see a functional test which is making use of this as a filter.  Maybe we use it for sorting?

Reply:
We are allowing our user to sort on the P50/95/99 on the rule list table, that's the reason why it need to be indexed